.github/workflows/elixir.yml

```diff
@@ -58,11 +58,9 @@ jobs:
             ${{ runner.os }}-${{ steps.setup-beam.outputs.elixir-version }}-
             ${{ runner.os }}-
       - name: Install Dependencies
-        run: mix deps.get --only $MIX_ENV
-      - name: Compile Dependencies
-        run: mix deps.compile --skip-umbrella-children
+        run: mix deps.get
       - name: Compile Application
-        run: mix compile --warnings-as-errors
```

Comment:
This is caught in the static analysis job

Reply:
This is mostly needed not to write a cache if there is a compilation error, so let's keep it.

Reply:
Fixed